### ISSUE TITLE
Dashed Outline on Buttons on iPad

### ISSUE THREAD

**Issue by damian-przygodzki** [equinor/design-system#3776]
**Describe the bug**

Dashed outlines shows after touch the button on iPad.

![Image](https://github.com/user-attachments/assets/7568197d-2050-4bff-966d-5358159ad338)

**Steps to reproduce the bug**

1. Trigger onClick with touch event

**Expected behavior**

Focus outline not visible after touching

**Specifications**

- Version: 0.44
- Browser: Edge
- OS: iOS

**Comment by oddvernes** [comment]
This looks to be a [bug](https://bugs.webkit.org/show_bug.cgi?id=247416) in webkit.
